src/porepy/numerics/nonlinear/nonlinear_solvers.py

```diff
@@ -71,14 +66,15 @@ def solve(self, model):
 
         return error_norm, is_converged, iteration_counter
 
-    def iteration(self, model, lin_tol):
+    def iteration(self, model):
         """A single Newton iteration.
 
         Right now, this is a single line, however, we keep it as a separate function
         to prepare for possible future introduction of more advanced schemes.
         """
```

Comment:
Please add description of arguments and returned value.

Reply:
While in principle always in favour of updating documentation, I respectfully don't think it's the PR creator's responsibility to update documenation in all files touched in that PR.